bin/blackmagic.py

```diff
@@ -481,11 +481,14 @@ def get_built_images(self, request):
         self._init_mongodb()
         user = User.objects.get(id=self.user_id)
         firmwares = Firmware.objects.filter(user=user) \
-                                    .filter(status=Firmware.DONE) \
+                                    .exclude(status=Firmware.INITIALIZED) \
                                     .order_by('-started_at')
         result = []
         for firmware in firmwares:
             f = {'name': firmware.name}
+            f['status'] = firmware.status
+            f['delete'] = firmware.status in (Firmware.DONE, Firmware.FAILED)
```

Comment:
@eugulixes I'd replace 'delete' with 'can_delete' and 'download' with 'can_donwload'